src/Infrastructure/Mail/DonationConfirmationMailSubjectRenderer.php

```diff
@@ -20,8 +20,12 @@ public function __construct(
 	 * @param array<string, mixed> $templateArguments
 	 */
 	public function render( array $templateArguments = [] ): string {
-		if ( $templateArguments['donation']['paymentType'] === PaymentType::BankTransfer->value ) {
-			return $this->translator->trans( $this->bankTransferSubjectKey );
+		if (
+			is_array( $templateArguments['donation'] ) &&
```

Comment:
You can probably also drop the first check from here

Reply:
@Abban If I do as you suggest then the initial error comes back:

 ------ --------------------------------------------------------------------- 
  Line   src/Infrastructure/Mail/DonationConfirmationMailSubjectRenderer.php  
 ------ --------------------------------------------------------------------- 
  24     Cannot access offset 'paymentType' on mixed.                         
         🪪  offsetAccess.nonOffsetAccessible                                 
 ------ ---------------------------------------------------------------------
 
 What should be the next step then?